fe1-web/src/features/wallet/objects/__mocks__/mockWallet.ts

```diff
@@ -0,0 +1,43 @@
+import { mockLao, mockRollCallState } from '__tests__/utils/TestUtils';
+
+import { Hash } from '../../../../core/objects';
+import { dispatch } from '../../../../core/redux';
+import { addEvent, removeEvent } from '../../../events/reducer';
+import { connectToLao, disconnectFromLao } from '../../../lao/reducer';
+import { RollCall } from '../../../rollCall/objects';
+import { generateToken } from '../Token';
+
+const createRollCall = (Id: string, mockAttendees: string[]) => {
+  return RollCall.fromState({
+    ...mockRollCallState,
+    id: Id,
+    name: Id,
+    attendees: [...mockRollCallState.attendees, ...mockAttendees],
+  });
+};
+const mockRCID0: string = '0mock';
+const mockRCID1: string = '1mock';
+const hashMock0 = new Hash(mockRCID0);
+const hashMock1 = new Hash(mockRCID1);
+/*
+ * Generates a mock state with some mock popTokens
+ */
+export function useMockWalletState() {
```

Comment:
The `use` prefix is usually used for react hooks, this does not seem to be one or am I missing something?